### PR TITLE
Batch removal of duplicated test urls

### PR DESCRIPTION
**Pull request by cschanaj**
@Hainish A dirty command is used to remove duplicated test urls found; `log.txt` is the Travis output of the failing test in #14660 

Block #14660

```bash
TRAVIS_REPLACE='s/^ERROR \([^:]*\): Duplicated test URL found \(.*\)$/\1 \2/g'

grep ^ERROR ~/log.txt  | tr -d '\r' | sed "$TRAVIS_REPLACE" | sort -u | while read FILE HOST; 
do 
    REGEX_ESCAPED_HOST=`python3.6 -c "import re; print(re.escape('$HOST'))"`;  
    sed -i "0,/<test.*\"$REGEX_ESCAPED_HOST\"/{//d;}" $FILE; 
done
```

See https://stackoverflow.com/questions/23696871/how-to-remove-only-the-first-occurrence-of-a-line-in-a-file-using-sed

### TODO

16 rulesets failed `fetch-test`

- [x] Destructoid.xml (3672bc6)
- [x] Hdfcbank.com.xml (59c6493)
- [x] Komoona.com.xml (115b6d7)
- [x] PropellerAds.xml (8888416) 
- [x] WP-Engine.xml (fddbf28)
- [x] Wemfbox.ch.xml (f0ad991)
- [x] cloud66.com.xml (f8e49bf)
- [x] greenhouseci.com.xml (fc55268)
- [ ] JD.com-Problematic.xml
- [ ] espivblogs.net.xml
- [ ] Twenga.com.xml
- [ ] Fenland.gov.uk.xml
- [ ] Wandoujia.xml
- [ ] luosimao.com.xml
- [ ] tanx.com.xml

6 rulesets failed `validations-test`
- [x] Imgs_Mail.ru.xml (d74a705)
- [x] Threema.xml (895c094)
- [x] Twenga.com.xml (c79806c)
- [x] Wemfbox.ch.xml (9f2573e)
- [x] Fenland.gov.uk.xml (f1075ff)
- [x] Komoona.com.xml (870e2d7)

2 rulesets having more than 1 duplicates
- [x] CloudAtCost.com.xml (628b7f5)
- [x] westberks.gov.uk.xml (e59ff8b)

### CODE REVIEW

File: src/chrome/content/rules/CloudAtCost.com.xml [hunk 1]
```diff
@@ -15,14 +15,12 @@
...
 
-        <test url="http://panel.cloudatcost.com/" />
         <test url="http://panel.cloudatcost.com:12345/" />
         <test url="http://panel.cloudatcost.com:59848/console.html" />
 
     <rule from="^http:" to="https:" />
 
         <test url="http://panel.cloudatcost.com/" />
-        <test url="http://members.cloudatcost.com/" />
         <test url="http://panel.cloudatcost.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/CloudAtCost.com.xml]: Here are even more duplicates.

**Comment by J0WI** [comment]
Rulesets that don't pass our tests anyway should be excluded in this PR. Maybe some of them are already whitelisted and the hash needs to be updated. I think just updating the hash would be fine here.

**Comment by cschanaj** [comment]
@J0WI thank for pointing out that some rulesets having more than 1 duplicates. I have updated the top comment and I will go through them later.

**Comment by cschanaj** [comment]
@Hainish @J0WI Travis is finally passing now.

**Comment by J0WI** [comment]
`squat.gr` and `mobilityrpmcc.hdfcbank.com` WFM, all other changes LGTM.

**Comment by J0WI** [comment]
Finally all tests passed. Thanks for fixing all those issues!

**Pull request merged**